controller_joint_names: ['', 'right_wheel_joint', 'left_wheel_joint', 'joint_1', 'joint_2', 'joint_3', 'joint_4', ]
robot_do_an:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
  # Effort Controllers -----------------------------------------
  left_wheel_controller:
    type: effort_controllers/JointVelocityController
    joint: left_wheel_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  right_wheel_controller:
    type: effort_controllers/JointVelocityController
    joint: right_wheel_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}